projects/mac_ai/testing/podman_machine.py

```
@@ -4,10 +4,10 @@
 import json, yaml
 
 from projects.core.library import env, config, run, configure_logging, export
-import remote_access, podman
+import remote_access, podman, prepare_virglrenderer, prepare_llama_cpp
 
 
-def _run(base_work_dir, cmd, check=True, capture_stdout=False, machine=True, get_command=False):
+def _run(base_work_dir, cmd, env={}, check=True, capture_stdout=False, machine=True, get_command=False, print_cmd=False):
```

Pylint (3.3.7)

[refactor] Too many arguments (8/5)

(R0913)